### ISSUE TITLE
About Novel Classes during Training

### ISSUE THREAD

**Issue by kinredon** [VinAIResearch/LP-OVOD#2]
Thanks for your work!

This work achieved 40.5 mAP on COCO without external datasets or knowing novel classes during training. However I found that the Linear Probing on Novel Classes would indeed use novel classes text embedding. Is my understanding correct?

**Comment by truong-v** [comment]
Yes, it is correct. However the linear probing and fine-tuning time is insignificant compare to base model training time.

**Comment by davidsonkillian** [comment]
During the training process of a neural network, such as in deep learning models like GPT (Generative Pre-trained Transformer), the concept of "novel classes" typically does not directly apply. However, I'll explain how novel classes or new concepts might relate to training in deep learning contexts[ areej shah novels](https://areejshahnovel.com/).

Fine-Tuning: In fine-tuning, a pretrained model is further trained on a specific task or dataset. If the task involves identifying or classifying objects, such as in computer vision, encountering novel classes can be relevant. For example, if a model trained to recognize cats and dogs is later fine-tuned to recognize birds, the classes of birds would be novel to the model during its original training.

**Comment by kunpeng337** [comment]
> Yes, it is correct. However the linear probing and fine-tuning time is insignificant compare to base model training time.是的，这是正确的。但是，与基本模型训练时间相比，线性探测和微调时间微不足道。

Are you answering what seems to be a pre-training and fine-tuning time question? I was wondering if you could tell me if linear probing of Novel Classes does use text embedding of novel classes in the absence of external data sets or without knowledge of new classes during training is in conflict with the premise of not knowing new classes mentioned in the open vocabulary?

**Comment by kunpeng337** [comment]
Since the new categories mentioned in open vocabulary goal detection are not known during the training, is the text embedding using novel classes mentioned in the paper a text embedding using the text of the new classes generated?
